### ISSUE TITLE
Small details in Transe

### ISSUE THREAD

**Issue by JiyaoWei** [malllabiisc/CompGCN#18]
r	= self.init_rel if self.p.score_func != 'transe' else **torch.cat([self.init_rel, -self.init_rel], dim=0)**
why cat them when use ‘transe’？

cheers

**Comment by soumyasanyal** [comment]
For Transe variant of the CompGCN model, we found that setting the prior `rel_inv = -rel` helps the model performance (also seemed logical to do so). Its not directly clear how to use such priors for other models, hence we didn't try for them. You can check [this line](https://github.com/malllabiisc/CompGCN/blob/3b06f5fec42526faa81afc158df9e64a8382982c/model/models.py#L29) to understand that we initialize with half the embedding size for Transe variant and later concat with the negative vector.
